playbooks/byo/openshift-cluster/cluster_hosts.yml

```diff
@@ -5,6 +5,10 @@ g_lb_hosts:     "{{ groups.lb | default([]) }}"
 
 g_master_hosts: "{{ groups.masters | default([]) }}"
 
+g_new_master_hosts: "{{ groups.new_masters | default([]) }}"
+
+g_all_master_hosts: "{{ g_master_hosts | union(g_new_master_hosts) | default([]) }}"
+
```

Comment:
make sure these groups are added to the cluster_hosts.yaml for aws, gce, libvirt and openstack